data/templates/unattend_server2012.xml

```diff
@@ -109,12 +109,20 @@
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
                     <Order>2</Order>
+                    <% if( firewallDisable ) { %>
+                    <CommandLine>powershell -Command "Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False" </CommandLine>

```

Comment:
I think you need this line and the next line to be enclosed with <% %>, otherwise the firewall is going to be disabled for all installs.